packages/data-service-generator/src/admin/entity/entity-title-component/entity-title-component.template.tsx

```diff
@@ -7,5 +7,5 @@ declare const ENTITY_TITLE_FIELD_NAME = "title";
 export const ENTITY_TITLE_FIELD_NAME_ID = ENTITY_TITLE_FIELD_NAME;
 
 export const ENTITY_TITLE = (record: ENTITY): string => {
-  return record.ENTITY_TITLE_FIELD || String(record.id);
+  return record.ENTITY_TITLE_FIELD?.toString() || String(record.id);
```

Comment:
in order to update the snapshot testing with the new expectation, you can run `npx nx update-snapshot data-service-generator` @Shurtu-gal

Reply:
Thanks

Reply:
Done